requests/packages/__init__.py

```diff
@@ -34,9 +34,3 @@
 except ImportError:
     import chardet
     sys.modules['%s.chardet' % __name__] = chardet
-
-try:
-    from . import idna
-except ImportError:
-    import idna
-    sys.modules['%s.idna' % __name__] = idna
```

Comment:
@Lukasa : In order to prevent the same behavior, I think we should still be using the logic i.e. `import` with `try..except` for `idna`. But remove this code block from `__init__.py` to the place where we are importing `idna` i.e. within `_get_idna_encoded_host()` of my PR in `models.py`. What are your thoughts on this?